scraper.rb

```diff
@@ -64,7 +64,30 @@ def scrape_term(url)
     }
     data[:image] = URI.join(url, data[:image]).to_s unless data[:image].to_s.empty?
     data[:end_date] = date_from(data[:notes]) if data[:notes].to_s.include? 'Resigned on '
+    data[:end_date] ||= term[:end_date]
     data[:start_date] = date_from(data[:notes]) if data[:notes].to_s.include? 'NMP term effective '
+    data[:start_date] ||= term[:start_date]
```

Comment:
I'm not sure why you're adding this (or the equivalent at line 67). It doesn't seem relevant to this change, isn't really necessary, and introduces the potential for a very nasty and subtle bug upstream much later if we discover that the start or end date for a term was actually incorrect.